src/shared/containers/LibraryCardForm/LibraryCardForm.jsx

```diff
@@ -42,17 +43,14 @@ class LibraryCardForm extends React.Component {
     this.handleOnBlur = this.handleOnBlur.bind(this);
   }
 
+
   componentDidMount() {
     const csrfToken = this.getMetaTagContent('name=csrf-token');
 
     if (csrfToken) {
-      this.setState({ csrfToken });
-    }
+      this.state.patronFields.location = this.getUrlParameter('form_type') || 'nyc';
```

Comment:
This should be part of the `setState` function instead of directly assigning it.

Reply:
Thanks! I made the update.